package.json

```diff
@@ -10,7 +10,7 @@
     "setup": "pnpm --filter \"./tools/**\" install",
     "docs:publish": "pnpm exec scripts/publish-docs.sh",
     "docs:build": "pnpm exec scripts/build-docs.sh",
-    "docs:watch": "chokidar packages/*/docs/*.md -c 'pnpm docs:build' --throttle 2000",
```

Comment:
I took this off because with throttle and debounce it seemed to rebuild like twice and then stop. Anyway it seems to be fine